libs/gi/ui/src/components/artifact/ArtifactCard.tsx

```diff
@@ -529,12 +544,21 @@ function SubstatDisplay({
     <Box display="flex" gap={1} alignContent="center">
       <Typography
         sx={{ flexGrow: 1 }}
-        color={numRolls ? `${rollColor}.main` : 'error.main'}
+        color={
+          numRolls
+            ? `${rollColor}.main`
+            : !isActiveStat
+              ? 'secondary'
+              : 'error.main'
+        }
         component="span"
       >
         <StatIcon statKey={stat.key} iconProps={iconInlineProps} />{' '}
         {tk(stat.key)}
```

Comment:
```suggestion
        {tk(`statKey_gen:${stat.key}`)}
```
youll have to update all current uses of tk if you do the change on 506